openslides/utils/cache.py

```diff
@@ -252,7 +269,7 @@ def use_redis_cache():
     return isinstance(caches['default'], RedisCache)
 
 
-def get_redis_connection():
+def get_redis_connection() -> Any:
```

Comment:
Hm. Is it possible to set the correct type easily?

Reply:
I don't know what type it is.